imp_act/environments/print_episode.py

```diff
@@ -3,23 +3,60 @@
 from imp_act import make
 
 
+def do_nothing_policy(observation):
+    return [[0] * len(e) for e in observation["edge_observations"]]
+
+def fail_replace_policy(observation):
+    actions = []
+    for edge in observation["edge_observations"]:
+        edge_action = []
+        for segment in edge:
+            if segment >= 3:
+                edge_action.append(3)
+            else:
+                edge_action.append(0)
+        actions.append(edge_action) 
+    return actions
+
+def heuristic_policy(observation):
+    actions = []
+    current_time = observation["time_step"]
+    for edge in observation["edge_observations"]:
+        edge_actions = []
+        for segment in edge:
+            if segment >= 5:
+                edge_actions.append(4)  # Reconstruction
+            elif segment >= 5:
```

Comment:
Minor comment: Since we only have 5 states {0,1,2,3,4} `segment >= 5` can be removed?

Reply:
Well, this is just the humble heuristic 😄 
I would leave it in, and we can change the values to test different policies.